### DISCUSSION TITLE
MacBook Pro 16 and charge limiter

### DISCUSSION THREAD

**Discussion by Barbas70** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1375] [Q&A] [answered]
Buongiorno,
I just bought a MacBook Pro 16 M4, it comes with a 140W USB-C Power Adapter.
In the office I will use it with a docking station that provides 65W for charging.

I wanted to know what happens if I set the charge limit to 80% when it is connected to 65W and maybe the MacBook is performing heavy processing and needs more than 65W.

Will he still use the battery or how will he behave?

I'm waiting to understand whether to proceed with the purchase of the pro version

Thank you

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @Barbas70,

Yes, when your MacBook needs more than 65 W the Battery will provide the rest of the power.
You can even keep track of where the energy is coming from with AlDente Pros [Power Flow feature](https://apphousekitchen.com/feature-explanation-power-flow/).

In general, the 140 W of the Apple power supply is mainly for fast charging the battery. Without charging the battery, the maximum load of the MacBook will be much lower than 140 W. Let's say the maximum load of the MacBook is somewhere around 90 W, which means that the battery needs to provide a maximum of ca. 25 W with a power supply of 65 W.

Best,
Matthias

**Reply by Barbas70** [reply]
Hi,

Thank you for your support!

Have a nice day
